deploy/helm/listener-operator/values.yaml

```diff
@@ -46,10 +46,7 @@ podAnnotations: {}
 podSecurityContext: {}
   # fsGroup: 2000
 
-securityContext:
-  # listener-operator requires root permissions
-  runAsUser: 0
-  privileged: true
```

Comment:
We can get rid of privileged mode, but we still need to run as root to have the right write permissions to the CSI volumes. :/